### PR TITLE
Ubiquiti Edgepower OS definition

### PR DESCRIPTION
**Pull request by jepke**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [ ] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/polling/os/edgeos-ep.inc.php [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?php
+
+$osdata = snmp_get_multi_oid($device, ['ubntModel.0', 'ubntVersion.0'], '-OQUs', 'UBNT-EdgeMAX-MIB');
+$hardware = 'Ubiquiti ' . $osdata['ubntModel.0'];
+$version = $osdata['ubntVerwion.0'];
```
Comment (PipoCanaja) [inline includes/polling/os/edgeos-ep.inc.php]: You have a typo here

**Comment by murrant** [comment]
I suppose we can merge this as-is, I really hope we don't have to merge it later.  That is a pain for a lot of people.

**Comment by jepke** [comment]
image

Supporting image for overview tab

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-62-release-changelog-march-2020/11423/1

**Pull request merged**